plugins/ata/fu-self-test.c

```diff
@@ -87,6 +87,7 @@ fu_ata_oui_func(void)
 int
 main(int argc, char **argv)
 {
+	g_setenv("FWUPD_DATADIR_QUIRKS", PLUGINSOURCEDIR, TRUE);
```

Comment:
Shouldn't we be setting this directly in the self test env={} dictionary in the meson file?